src/Annotation/ApiResource.php

```diff
@@ -251,19 +270,38 @@ final class ApiResource
     /**
      * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
      *
+     * @var string|false
+     */
+    private $input;
```

Comment:
This one is https://api-platform.com/docs/core/dto/#specifying-an-input-or-an-output-data-representation